python/HeaderService/hutils.py

```diff
@@ -192,13 +194,11 @@ def get_date_utc(timeStamp=None, format='isot'):
 
     """
 
-    from astropy.time import Time
-    from datetime import datetime
     if timeStamp is None:
-        utc_time = (datetime.utcnow()).isoformat()
+        utc_time = (datetime.datetime.utcnow()).isoformat()
```

Comment:
Avoid the extra stringification by doing:
```
t = astropy.time.Time.now()
```

Reply:
Done too.